addons/api/fnc_getAllRadiosByType.sqf

```diff
@@ -33,11 +31,4 @@ if (_array isEqualType objNull) then {
     };
 };
 
-{
-    private _radioId = _x;
-    if ([_radioId, _radioType] call FUNC(isKindOf)) then {
-        _ret pushBackUnique _radioId;
-    };
-} forEach _array;
-
-_ret
+_array select {[_x, _radioType] call FUNC(isKindOf)}
```

Comment:
This doesn't filter for unique items, but I guess this is fine as that shouldn't be possible to be had (without other issues).

Reply:
this can only happen if a user adds in duplicated items in the beginning.
we could run an arrayIntersect on the return, but I don't know if that is really worth it.

Reply:
I think running `arrayIntersect` would be fine.